Adtangir/gdap documentation (#15965)
microsoftgraph/microsoft-graph-docs: api-reference/beta/resources/delegatedadminrelationshiprequest.md

---
title: "delegatedAdminRelationshipRequest resource type"

> description: "Represents a request specific to a delegated admin relationship between a partner and customer."

Reviewer: This description isn't completely clear when I look at the values of the two enums

author: "smrtsec"
ms.localizationpriority: medium
ms.prod: "gdap"
doc_type: resourcePageType
---

# delegatedAdminRelationshipRequest resource type

Namespace: microsoft.graph

[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]

> Represents a request specific to a delegated admin relationship between a partner and customer.

Reviewer: Please also make the change here

|[Create delegatedAdminRelationshipRequest](../api/delegatedadminrelationship-post-requests.md)|[delegatedAdminRelationshipRequest](../resources/delegatedadminrelationshiprequest.md)|Create a new [delegatedAdminRelationshipRequest](../resources/delegatedadminrelationshiprequest.md) object.|
|[Get delegatedAdminRelationshipRequest](../api/delegatedadminrelationshiprequest-get.md)|[delegatedAdminRelationshipRequest](../resources/delegatedadminrelationshiprequest.md)|Read the properties and relationships of a [delegatedAdminRelationshipRequest](../resources/delegatedadminrelationshiprequest.md) object.|
|[Update delegatedAdminRelationshipRequest](../api/delegatedadminrelationshiprequest-update.md)|[delegatedAdminRelationshipRequest](../resources/delegatedadminrelationshiprequest.md)|Update the properties of a [delegatedAdminRelationshipRequest](../resources/delegatedadminrelationshiprequest.md) object.|
|[Delete delegatedAdminRelationshipRequest](../api/delegatedadminrelationshiprequest-delete.md)|None|Deletes a [delegatedAdminRelationshipRequest](../resources/delegatedadminrelationshiprequest.md) object.|

## Properties
|Property|Type|Description|
|:---|:---|:---|

> |action|String|The action to be performed on the delegated admin relationship.|
> |createdDateTime|DateTimeOffset|The date and time (ISO 8601) at which the relationship request was created in UTC. This is set by the system and cannot be set by the caller.|
> |id|String|The unique identifier of the relationship request. This is set by the system and cannot be set by the caller.|
> |lastModifiedDateTime|DateTimeOffset|The date and time (ISO 8601) at which this relationship request was last modified in UTC. This is set by the system and cannot be set by the caller.|
> |status|String|The status of the request.|

Reviewer: status and action are enums. Please copy over the complete list of members from the doc stub.
```suggestion
|action|String|The action to be performed on the delegated admin relationship.|
|createdDateTime|DateTimeOffset|The date and time in ISO 8601 when the relationship request was created and is always in UTC. Read-only.|
|id|String|The unique identifier of the relationship request. Read-only.|
|lastModifiedDateTime|DateTimeOffset|The date and time in ISO 8601 when this relationship request was last modified and is always in UTC. Read-only.|
|status|String|The status of the request.|
```
